README.md

```diff
@@ -31,27 +31,32 @@ git clone git@github.com:openjdk/mobile.git
 ```
 
 ### Configure
-Modify the configure script below so that it has the correct paths to your JDK24 directory,
-the unzipped 'support' directory, and the correct location of the iPhoneOS.platform.
+Modify the configure script below so that it has the correct paths to the unzipped 'support'
+directory, and the correct location of the iPhoneOS.platform.
 
 ```
 sh configure \
---with-conf-name=zero-ios-aarch64 \
 --disable-warnings-as-errors \
 --openjdk-target=aarch64-macos-ios \
---with-boot-jdk=/Users/abhinay/.sdkman/candidates/java/24.ea.29-open \
---with-jvm-variants=zero \
 --with-libffi-include=<support-dir-absolute-path>/libffi/include \
 --with-libffi-lib=<support-dir-absolute-path>/libffi/libs \
 --with-cups-include=<support-dir-absolute-path>/cups-2.3.6
```

Comment:
Configure arguments don't have to be absolute paths; configure will rewrite then to absolute path if they are relative.